mtb-ce-manifest-fv2.xml

```diff
@@ -1267,11 +1267,19 @@ iBeacon UUID data.
     <name>SGPIO Target Interface</name>
     <id>mtb-example-psoc6-smartio-sgpio-target</id>
     <category>Peripherals</category>
-    <uri>https://github.com/cypresssemiconductorco/mtb-example-psoc6-smartio-sgpio-target</uri>
+    <uri>https://github.com/Infineon/mtb-example-psoc6-smartio-sgpio-target</uri>
```

Comment:
For the repos transferred from cypresssemiconductorco, this should not be changed.